checkov/terraform/runner.py

```diff
@@ -116,47 +117,73 @@ def run(
         parsing_errors: dict[str, Exception] = {}
         self.load_external_checks(external_checks_dir)
         local_graph = None
-
+        all_graphs = []
```

Comment:
seems like you only use this variable at the end of this function, so I think it's better to move it there

Reply:
Yes it was inside at the beginning but then I got a bunch of errors like: `Local variable referenced before assignment`
The reason is we use this starting at line 195, so I have to declare it out of the inner scope